File: Python/Program/SSD.py
from Komponen import Komponen

class SSD(Komponen):
    def __init__(self, kapasitas_gb, merk, nama):
        Komponen.__init__(self, merk, nama) 
        self.kapasitas_gb = kapasitas_gb

    def get_kapasitas_gb(self):
        return self.kapasitas_gb

File: C++/Program/SSD.cpp
#pragma once
#include <iostream>
#include <string>
#include "Komponen.cpp"

using namespace std;

class SSD : public Komponen
{
private:
    int kapasitasGB; 
public:
    SSD()
    {

    } 

    SSD(int kapasitasGB, string merk, string nama) : Komponen(merk, nama)
    {
        this->kapasitasGB = kapasitasGB;
    }

    void setKapasitasGB(int kapasitasGB)
    {
        this->kapasitasGB = kapasitasGB;
    }

    int getKapasitasGB()
    {
        return kapasitasGB;
    }

    ~SSD() {}
};